content/intermediate/250_cloudwatch_container_insights/wraup.md

```diff
@@ -24,4 +24,8 @@ If you are completely done with EKSWorkshop, run the following command to clean
 eksctl delete cluster --name=eksworkshop-eksctl
 ```
 
-## Thank you for using CloudWatch Container Insights! 
+{{% notice tip %}}
```

Comment:
I don't think we should be cleaning up the cluster inside of the cloudwatch chapter... can you remove this delete cluster instruction?

Reply:
I just updated the PR based on your feedback